components/project.py

```diff
@@ -25,3 +25,36 @@ class LogWidget(BaseElement):
 
     # Group Locators
     log_items = GroupLocator(By.CSS_SELECTOR, '#logFeed .db-activity-item')
+
+
+class CreateRegistrationModal(BaseElement):
+    modal_window = Locator(By.CSS_SELECTOR, '[data-test-new-registration-modal]')
+    cancel_button = Locator(
+        By.CSS_SELECTOR, 'button[data-test-new-registration-modal-cancel-button]'
+    )
+    create_draft_button = Locator(
+        By.CSS_SELECTOR, 'button[data-test-new-registration-modal-create-draft-button]'
+    )
+
+    # Group Locators
+    schema_list = GroupLocator(
+        By.CSS_SELECTOR, '[data-test-new-registration-modal-schema]'
+    )
+
+    def get_schema_names_list(self):
+        """Returns the schema names from the schema list"""
+        names_list = []
+        for schema in self.schema_list:
+            names_list.append(schema.text)
+        return names_list
```

Comment:
Nitpick / golfing: the above code can be simplified down to: `return [schema.text for schema in self.schema_list]`

Reply:
Fixed! I like it. Definitely more streamlined. I'm still getting used to all the python syntax and shortcuts.